common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
```

Comment:
@cptvitamin Thanks for the help. To clarify, is this your recommendation?

```html
<span class="clarification" tabindex="0" role="note" aria-label="Clarification">
	<i data-tooltip="${clarification | h}" data-tooltip-show-on-click="true"
            class="fa fa-info-circle" aria-hidden="true"></i>
    <span class="sr">(${clarification})</span>
</span>
```

Will the new `role` field handle the hide/show or do I need to add new JS to hide/show the inner span.sr on focus?